### ISSUE TITLE
Vertical sliders?

### ISSUE THREAD

**Issue by the-t-in-rtf** [dinukadesilva/music-ctrls#110]
Hi, there.  Thanks for your work on this project, I am already starting to use it in my own work with Flocking.  It looks great and works well!

I saw in [another issue](https://github.com/dinukadesilva/music-ctrls/issues/58) that there were mockups of vertical sliders, but I don't see any examples in the demo or tests.  Is that implemented?  Please advise.

**Comment by dinukadesilva** [comment]
Yes @the-t-in-rtf ...... still we haven't implemented the vertical slider... we have implemented only the horizontal slider yet.

**Comment by the-t-in-rtf** [comment]
It would be useful to represent things like MIDI modulation and pitch bend controls.  I will give it some thought, I plan to submit a PR to add this in a week or so.

**Comment by dinukadesilva** [comment]
Great..... 

Happy to help you @the-t-in-rtf
